test_runner/regress/test_download_extensions.py

```diff
@@ -84,10 +44,37 @@ def upload_files(env):
 @pytest.mark.parametrize("remote_storage_kind", available_s3_storages())
 @pytest.mark.skip(reason="https://github.com/neondatabase/neon/issues/4949")
 def test_remote_extensions(
-    neon_env_builder: NeonEnvBuilder,
     remote_storage_kind: RemoteStorageKind,
     pg_version: PgVersion,
+    pytestconfig: Config,
+    test_output_dir: Path,
+    port_distributor: PortDistributor,
+    mock_s3_server: MockS3Server,
+    neon_binpath: Path,
+    pg_distrib_dir: Path,
+    default_broker: NeonBroker,
+    run_id: uuid.UUID,
 ):
+    alt_dir = test_output_dir / "pg_install"
+    log.warn(
+        f"Copying {pg_distrib_dir} to {alt_dir}. \
+            It is ~1.7GB so you might want to delete it from your test_output"
+    )
```

Comment:
```suggestion
    log.warn(f"Copying {pg_distrib_dir} to {alt_dir}")
```

After the test, we delete almost everything from `test_output_dir` (see `def cleanup_local_storage`), so there's no need to do it manually

Reply:
oh nice!